canonical-web-and-design/tools.demos.haus
# Fix Ubuntu logo dimensions

## Done

Update the dimensions on the Ubuntu logo SVG

## QA

- Check out this feature branch
- Run the site using the command `./run serve`
- View the site locally in your web browser at: http://0.0.0.0:8057
- Check that the Ubuntu logo looks correct and isn't squashed as in the screenshot below

## Screenshots
![Screenshot from 2020-07-10 13-47-46](https://user-images.githubusercontent.com/501889/87156492-a49ee500-c2b4-11ea-9f49-3da5fca6945f.png)
